packages/flutter_tools/test/general.shard/devtools_launcher_test.dart

```diff
@@ -153,3 +237,7 @@ void main() {
     expect(logger.errorText, contains('Failed to launch DevTools: ProcessException'));
   });
 }
+
+
+class MockPlatform extends Mock implements Platform {}
+class MockPersistentToolState extends Mock implements PersistentToolState {}
```

Comment:
There is a test version of this class that should be preferred over the mock:  `PersistentToolState.test(...)`

Reply:
done